stop_and_wait_plugin/src/stop_and_wait_plugin.cpp

```diff
@@ -216,6 +216,14 @@ std::vector<carma_planning_msgs::msg::TrajectoryPlanPoint> StopandWait::trajecto
   for (size_t i = 0; i < points.size(); i++)
   {
     carma_planning_msgs::msg::TrajectoryPlanPoint tpp;
+
+    if (times[i] != 0 && !std::isnormal(times[i]) && i != 0)
+    {  // If the time
+      RCLCPP_WARN_STREAM(rclcpp::get_logger("stop_and_wait_plugin"),"Detected non-normal (nan, inf, etc.) time."
```

Comment:
Are these times or speeds?

Reply:
Times. The time is going to `inf` because speed is zero and dt = dx / dv = dx / 0 = `inf`

Reply:
EDIT: Correction, they are `nan` not `inf` because dx and dv both 0 where 0/0 = `nan` is happening